src/contribute/contribution-guide.md

```diff
@@ -201,10 +201,12 @@ For contributions to the Java code, run unit tests locally via Maven. Alternativ
 Once the initial code is complete and the tests pass, it’s time to start the code review process. We review and discuss all code, no matter who authors it. It’s a great way to build community, since you can learn from other developers, and they become familiar with your contribution. It also builds a strong project by encouraging a high quality bar and keeping code consistent throughout the project.
 
 ### Create a pull request
-Organize your commits to make your reviewer’s job easier. Use the following command to re-order, squash, edit, or change description of individual commits.
+Organize your commits to make your reviewer’s job easier. Reviewers normally prefer a minimum number of commits that break up a PR into logical steps. For most PRs, you'll squash your changes down to 1 commit. You can use the following command to re-order, squash, edit, or change description of individual commits.
```

Comment:
Points:
* multiple small PRs, instead of a large PR
* within a PR, a relatively small number of commits that break the problem into logical steps

Everwhere "PR" -> pull request.